code/game/objects/structures/extinguisher.dm

```diff
@@ -48,6 +51,27 @@
 		opened = !opened
 	update_icon()
 
+/obj/structure/extinguisher_cabinet/attackby(obj/item/W as obj, mob/user as mob, params)
+	if(opened)
+		if(istype(W, /obj/item/weldingtool))
+			var/obj/item/weldingtool/WT = W
+			if(!WT.remove_fuel(0, user))
+				to_chat(user, "<span class='notice'>You need more welding fuel to complete this task.</span>")
+				return
+			to_chat(user, "<span class='notice'>You begin cutting \the [src] apart...</span>")
+			playsound(loc, cutting_sound, 40, 1)
```

Comment:
You haven't defined `cutting_sound` as anything

Reply:
This should be `playsound(loc, W.usesound, 40, 1)`

The tool has the sound, not the structure (plus we are gonna remove that cutting_sound var anyways)